SPEC.md

```diff
@@ -327,7 +327,7 @@ Retrievable at `http://$AC_METADATA_URL/acMetadata/v1/apps/${ac_app_name}/`
 
 | Entry         | Description |
 |---------------|-------------|
-|annotations/   | A directory of metadata values on the entrypoint manifest.|
+|annotations/   | Annotations from image manifest merged with and overriden by app annotations from container runtime manifest.|
```

Comment:
remove "and overriden [sic] by" here, I think it is fine to just have it in the explanation below